.travis.yml

```diff
@@ -30,35 +30,34 @@ env:
         - CONDA_CHANNELS='astropy-ci-extras astropy'
 
     matrix:
-        # Make sure that egg_info works without dependencies
+
         - SETUP_CMD='egg_info'
-        # Try all python versions with the latest numpy
-        - SETUP_CMD='test'
 
 matrix:
     include:
 
-        # Do a coverage test in Python 2.
-        - python: 2.7
+        # Try Astropy/NumPy development versions. This requires them to be
+        # compiled during setup which takes some time.
+        - python: 3.5
+          env: ASTROPY_VERSION=development NUMPY_VERSION=dev
+
+        # Do coverage tests for both latest Python versions
+        - python: 3.5
           env: SETUP_CMD='test --coverage'
 
-        # Check for sphinx doc build warnings - we do this first because it
-        # may run for a long time
         - python: 2.7
-          env: SETUP_CMD='build_sphinx -w'
+          env: SETUP_CMD='test --coverage'
```

Comment:
If it was a minute I'd remove it again but it seems more like 10-20 seconds (or travis fluctuations):

- [2:50 (without coverage)](https://travis-ci.org/astropy/ccdproc/builds/172683391)
- [2:50 and 3:05 (with coverage)](https://travis-ci.org/astropy/ccdproc/builds/172698216)

`ccdproc` does not have as many lines as astropy. I think it will take quite a while until `coverage` significantly slows down the testing.